src/server/cmd/pachd/main.go

```diff
@@ -79,8 +78,7 @@ func init() {
 }
 
 func main() {
-	log.SetFormatter(logutil.FormatterFunc(logutil.Pretty))
-	maxprocs.Set(maxprocs.Logger(log.Printf))
```

Comment:
We shouldn't delete the call to maxprocs.Set.  This copies k8s limits on the container over to GOMAXPROCS, which is important for reasonable latency.

Reply:
oops, thanks!